database2/README.md

```diff
@@ -0,0 +1,77 @@
+database
+========
+
+[![Build Status](https://travis-ci.org/btcsuite/btcd.png?branch=master)]
+(https://travis-ci.org/btcsuite/btcd)
+
+Package database provides a block and metadata storage database.
+
+Please note that this package is intended to enable btcd to support different
+database backends and is not something that a client can directly access as only
+one entity can have the database open at a time (for most database backends),
+and that entity will be btcd.
+
+When a client wants programmatic access to the data provided by btcd, they'll
+likely want to use the [btcrpcclient](https://github.com/btcsuite/btcrpcclient)
+package which makes use of the [JSON-RPC API]
+(https://github.com/btcsuite/btcd/tree/master/docs/json_rpc_api.md).
+
+However, this package could be extremely useful for any applications requiring
+Bitcoin block storage capabilities.
+
+As of July 2015, there are over 365,000 blocks in the Bitcoin block chain and
+and over 76 million transactions (which turns out to be over 35GB of data).
+This package provides a database layer to store and retrieve this data in a
+simple and efficient manner.
+
+The default backend, ffldb, has a strong focus on speed, efficiency, and
+robustness.  It makes use leveldb for the metadata, flat files for block
```

Comment:
It makes use leveldb for the metadata 
--->
It makes use **of** leveldb for the metadata